src/Command/CreateClassCacheCommand.php

```diff
@@ -16,7 +16,14 @@
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 
+@trigger_error(
+    'The '.__NAMESPACE__.'\CreateClassCacheCommand class is deprecated since version 3.x and will be removed in 4.0.',
+    E_USER_DEPRECATED
+);
+
 /**
+ * NEXT_MAJOR: Remove this class.
```

Comment:
btw. do we need a `@deprecated` tag here? I am still confused about, NEXT_MAJOR, `@deprecated` ... tbh 😅

Reply:
`NEXT_MAJOR` is our tag, when we clean master for a major release we search for all `NEXT_MAJOR` and do something based on that. `@deprecate` basically says that you shouldn't use that and in phpstorm you get nice crossed line like: ~~CreateClassCacheCommand~~